components/spi/.tekton/pipelines-as-code.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: service-provider-integration-pac
+  namespace: service-provider-integration
```

Comment:
the namespace should be spi-system, or rather do not mention it here, so it will be managed by components/spi/kustomization.yaml

Reply:
Done, removed the namespace.